packages/documentation-v7/src/stories/utilities/background/background.docs.mdx

```diff
@@ -0,0 +1,82 @@
+import { Meta, Canvas } from '@storybook/blocks';
+import LinkTo from '@storybook/addon-links/react';
+import { BADGE } from '../../../../.storybook/constants';
+import { forEach } from '../../../utils/react';
+import { parse } from '../../../utils/sass-export';
+import { BackgroundContainer } from './background.blocks';
+import * as BackgroundDemos from './background.demo.stories';
+import scss from './background.module.scss';
+import './background.styles.scss';
+
+export const backgroundColors = parse(scss);
+
+<Meta
+  title="Utilities/Background"
+  parameters={{ badges: [BADGE.TODO] }}
+/>
+
+# Background
+
+<div className="lead">
```

Comment:
Nitpick: we can use class now instead of className

Reply:
Ah I got confused because I tried in the icons.docs.mdx, and it gets rendered (but still display a console error)